package.json

```diff
@@ -9,19 +9,20 @@
     "url": "git://github.com/shtylman/localtunnel.git"
   },
   "dependencies": {
-    "request": "2.65.0",
+    "request": "^2.68.0",
     "yargs": "3.29.0",
     "debug": "2.2.0",
     "openurl": "1.1.0"
   },
   "devDependencies": {
+    "snyk": "^1.13.3",
```

Comment:
As you can see below, I added `snyk test` to the test process, to help highlight whenever a new vulnerable dependency is added (or perhaps a new vulnerability is disclosed on current deps). This requires snyk as a devDependency.